src/Compat.jl

```diff
@@ -641,6 +641,15 @@ if v"1.10.0-" <= VERSION < v"1.10.0-DEV.360" || VERSION < v"1.9.0-beta3"
     splat(f) = Splat(f)
 end
 
+# https://github.com/JuliaLang/julia/pull/25085
+if VERSION < v"1.8.0-beta2.17"
```

Comment:
Strictly speaking, this should probably be:
```suggestion
if VERSION < v"1.8.0-beta2.17" || v"1.9.0-" <= VERSION < v"1.9.0-DEV.94
```
I doubt that'll make any difference in practice, though.